collectionchallenge.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+$db = new PDO('mysql:host=db; dbname=pokemon', 'root', 'password' );
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+$query = $db->prepare("SELECT `stats`.`name`, `stats`.`type1`, `stats`.`type2`, `stats`.`hp`, `stats`.`attack`, `stats`.`defense`, `stats`.`spAttack`, `stats`.`spDefense`, `stats`.`speed` FROM `stats`;");
+$query->execute();
+$result = $query->fetchAll();
+
+var_dump($result);
```

Comment:
not needed in final so maybe don't merge this

Reply:
Removed from file, there purely to show the database was correctly connected